explore/src/main/scala/explore/targeteditor/AladinContainer.scala

```diff
@@ -239,16 +244,7 @@ object AladinContainer {
       }
       // Use fov from aladin
       .useState(none[Fov])
-      // full screen trigger reflow
-      .useEffectWithDepsBy((props, _, _, _, _, _, _, _) => props.options.fullScreen)(
```

Comment:
Why is this no longer needed?

Reply:
Full screen triggers a redraw when the parent changes it. This only made the flashing effect worse by doing 2 extra repaints